CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+
+## [Unreleased]
+
+### Added
+- ISSUE-1 Add a memory server application
+- ISSUE-6 Produce simple docker images with JIB
```

Comment:
cannot we let github automatically handle this painful part for us?

Reply:
Can github do that? I have zero idea :)

Reply:
If you link each issue to a milestone, when you close the milestone you can have all the related issues.

Reply:
I will try to check but wondering in what kind of format that would end up too...

Reply:
do you mean this? https://github.com/github-changelog-generator/github-changelog-generator

Reply:
> If you link each issue to a milestone, when you close the milestone you can have all the related issues.

Is that a conf in github or something? I did not find anything regarding this. Just some plugins that you need to run manually I believe. Can you point to the source of this statement @rouazana ?

Reply:
What if, one day, we have to move out of github?

Won't we be happy to not be too dependant on it?

Reply:
I was simply thinking of that: https://docs.github.com/en/github/administering-a-repository/about-releases

Reply:
Did you ever try that? They do not say at all it generates automatically a changelog when you push the release button. When reading this: https://docs.github.com/en/github/administering-a-repository/managing-releases-in-a-repository I think it is something you still need to do manually at the end? 

I think you might be able to maybe automate something with using API https://docs.github.com/en/rest/reference/repos#releases and scripts to generate the changelog in the description with tools like I said above... But that's not out of the box, you need some work on it obviously

Reply:
> How can we include release details of apache/james-project with your proposition?

Good point, let's stick with the manual changelog then.